src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -21,7 +22,11 @@
  * The main LogicManager of the app.
  */
 public class LogicManager implements Logic {
-    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
+    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data. The following errors has occured: %s\n";
```

Comment:
```suggestion
    public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data. The following error has occured: %s\n";
```

Reply:
Not sure if the comment about `\n` at the end for `FILE_OPS_PERMISSION_ERROR_MESSAGE` also applies here